ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main__periodic-4.14.yaml

```diff
@@ -15,6 +15,10 @@ base_images:
     name: "4.14"
     namespace: ocp
     tag: openstack-tests
+  stolostron:
+    name: builder
+    namespace: stolostron
```

Comment:
wat?

Reply:
To run `make` and use Go, you need an image. It's not a problem for the installer & openstack-tets, they have their image. For periodic we don't have an image so I'm using https://github.com/stolostron/image-builder which is widely used in OCP CI.
I don't think it's a problem unless there is an image you prefer me to use.